pkgs/applications/emulators/blink/default.nix

```diff
@@ -11,6 +11,13 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-W7yL7Ut3MRygJhFGr+GIj/CK57MkuDTcenft8IvH7jU=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/jart/blink/commit/bfe4f5101da11964dd258f637eac779d066ec150.patch";
```

Comment:
This commit seems to not belong to any branch: https://github.com/jart/blink/commit/bfe4f5101da11964dd258f637eac779d066ec150
Is this not yet in master?

Reply:
Oh I took the commit from PR but they squash them on merge. Let me update that real quick